_config.yml

```diff
@@ -11,7 +11,7 @@ safe: false
 lsi: false
 url: https://docs.docker.com/v1.12
 keep_files: ["v1.4", "v1.5", "v1.6", "v1.7", "v1.8", "v1.9", "v1.10", "v1.11"]
-
+exclude: ["Dockerfile", "Gemfile", "Gemfile.lock", "hooks", "nginx-overrides.conf", "_scripts"]
 gems:
```

Comment:
This something we need to change?

```
Configuration file: /source/_config.yml
       Deprecation: The 'gems' configuration option has been renamed to 'plugins'. Please update your config file accordingly.
            Source: /source
       Destination: /site
 Incremental build: enabled
```

Reply:
Maybe eventually, but right now, changing it breaks Netlify. It's only a deprecation for now. Seems to be GH Pages being more opinionated than the generic Jekyll config.